sbt-dump-event-schema/src/sbt-test/sbt-dumpschema/simple/project/plugins.sbt

```
@@ -0,0 +1,5 @@
+sys.props.get("plugin.version") match {
+  case Some(x) => addSbtPlugin("org.virtuslab" % "sbt-dump-event-schema" % x)
+  case _ => sys.error("""|The system property 'plugin.version' is not defined.
+                         |Specify this property using the scriptedLaunchOpts -D.""".stripMargin)
```

`scriptedLaunchOpts -D.` is pretty unclear... you mean `-DscriptedLaunchOpts=...`?

---

Not really, I'm refering here to this:
https://github.com/VirtusLab/akka-safer-serializer/blob/6856608db929c9b4d0e71be1dbc0ce562df2b479/build.sbt#L98

---

Oh ok... maybe specify to:
```suggestion
                         |Specify this property using "-Dplugin.version=..." in `scriptedLaunchOpts` sbt setting""".stripMargin)
```